src/worker/worker-options.ts

```diff
@@ -16,4 +27,13 @@ export interface WorkerOptions {
    * @default false
    */
   async?: boolean
+  /**
+   * killBehavior dictates if your async unit ( worker/process ) will be deleted in case that a task is active on it.
```

Comment:
```suggestion
   * `killBehavior` dictates if your async unit (worker/process) will be deleted in case that a task is active on it.
```

Reply:
I am sorry I am doing my best to improve this